app/parse.py

```diff
@@ -8,5 +13,18 @@ class Course:
     duration: str
 
 
+def get_single_course(tag: Tag) -> Course:
+    return Course(
+        name=tag.find("h3").text,
+        short_description=tag.find("p", class_="mb-32").text,
+        duration=tag.find(
+            "p", class_="ProfessionCardTags_regularTag__yTc6K"
+        ).text
```

Comment:
Consider adding error handling for cases where the expected HTML elements (like 'h3', 'p.mb-32', or 'p.ProfessionCardTags_regularTag__yTc6K') are not found. This will prevent the code from breaking if the structure of the webpage changes or if some elements are missing.